src/config.rs

```diff
@@ -322,6 +322,8 @@ pub struct StaticAssetsMiddleware {
     /// Fallback page for a case when no asset exists (404). Useful for SPA
     /// (single page app) where routes are virtual.
     pub fallback: String,
+    /// Enable `precompressed_gzip`, will serve the .gz file if it exists
+    pub compress: bool,
```

Comment:
Changed.

Reply:
Thanks. Can you make it past tense: `precompressed`? (because the files are already precompressed, and we dont do any action, other than read-only decompression)

Reply:
Ah sure, changed.